packages/cdktf-cli/test/lib/cdktf-project.test.ts

```diff
@@ -211,6 +230,60 @@ describe("CdktfProject", () => {
       ]);
       return expect(eventTypes.includes("waiting for approval")).toBeFalsy();
     });
+
+    it("runs without terraform parallelism", async () => {
```

Comment:
The test seems to run with that flag, update the test name? Or am I missing something?

Reply:
```suggestion
    it("passes the terraform parallelism flag to terraform", async () => {
```

Reply:
Ah yeah, sorry, the wording here is confusing. I just used it because without parallelism would mean passing it a value of 1. By default parallelism is on (with a default value of 10). I like yours much better.

Reply:
It was actually yours (from the destroy test further down) 😁